src/AudioToolbox/MusicTrack.cs

```diff
@@ -263,7 +263,18 @@ public MusicPlayerStatus SetDestMidiEndpoint (MidiEndpoint endpoint)
 		{
 			return MusicTrackSetDestMIDIEndpoint (handle, endpoint == null ? MidiObject.InvalidRef : endpoint.MidiHandle);
 		}
-#endif // IOS
+
+		[DllImport (Constants.AudioToolboxLibrary)]
+		extern static /* OSStatus */ MusicPlayerStatus MusicTrackGetDestMIDIEndpoint (/* MusicTrack */ IntPtr inTrack, out MidiEndpointRef outEndpoint);
+
+		public MusicPlayerStatus GetDestMidiEndpoint (out MidiEndpoint outEndpoint)
+		{
+			MidiEndpointRef midiHandle; 
+			var reuslt = MusicTrackGetDestMIDIEndpoint (handle, out midiHandle);
+			outEndpoint = new MidiEndpoint (midiHandle);
```

Comment:
Maybe check if the P/Invoke succeeded before creating the MidiEndpoint instance.